### PR TITLE
Regenerate foreman_acd RPM spec based on latest template

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

**Comment by nadjaheitmann** [comment]
cc @sbernhard

File: packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec [hunk 1]
```diff
@@ -1,35 +1,28 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:4]: this variable is used a few times in the spec, but now that you're not doing any funny SCL things, you can just replace it with the "original" `_localstatedir`

File: packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
-BuildRequires: ruby
+Requires: ruby >= 2.5
+BuildRequires: ruby >= 2.5
 BuildRequires: rubygems-devel
 BuildArch: noarch
 Provides: foreman-plugin-%{plugin_name} = %{version}
-BuildRequires: rubygem(foreman_remote_execution) >= 7.0
-BuildRequires: rubygem(foreman-tasks) >= 7.0
+BuildRequires: rubygem(foreman_remote_execution) >= 3.3.0
+BuildRequires: rubygem(foreman-tasks) >= 0.10
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:25]: I guess some bumping inside the gemspec is in order? :)

Comment (nadjaheitmann) [reply packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:25]: :see_no_evil:

File: packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec (hunk 1 above)
Comment (nadjaheitmann) [reply packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:4]: @sbernhard Can you have a look, please :)

Comment (evgeni) [reply packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:4]: just do a `sed -i 's/_root_localstatedir/_localstatedir/g' packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec` ;)

**Comment by evgeni** [comment]
there is a `BuildRequires: npm(@theforeman/builder) < 11.0.0` in the spec that prevents it from building right now.

**Comment by nadjaheitmann** [comment]
> there is a `BuildRequires: npm(@theforeman/builder) < 11.0.0` in the spec that prevents it from building right now

:tired_face: Let me remove this...

File: packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec (hunk 1 above)
Comment (nadjaheitmann) [reply packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:4]: :woman_shrugging:

File: packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec (hunk 2 above)
Comment (nadjaheitmann) [reply packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:25]: Funny thing is that is says

>   s.add_dependency 'foreman_remote_execution', '>= 8.0'
     s.add_dependency 'foreman-tasks', '>= 7.0'

in https://github.com/ATIX-AG/foreman_acd/blob/master/foreman_acd.gemspec

Comment (evgeni) [reply packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec:25]: But not in https://github.com/ATIX-AG/foreman_acd/blob/v0.9.4/foreman_acd.gemspec, which is what we package here.

**Review by evgeni — APPROVED** [review]

**Pull request merged**